apps/docs/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "docs",
+  "scripts": {
+    "dev": "next dev -p 5000",
+    "build": "next build",
+    "start": "next start"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openstatusHQ/openstatus"
+  },
+  "homepage": "https://openstatus.dev",
+  "dependencies": {
+    "next": "^13.4.12",
+    "nextra": "latest",
```

Comment:
instead of latest can we specify the version to avoid some issue if there's an update ?

Reply:
Alright.

Reply:
`2.10.0` appears to be the latest version, I'll add that.